Framework/HistogramData/test/HistogramXTest.h

```diff
@@ -0,0 +1,20 @@
+#ifndef MANTID_HISTOGRAMDATA_HISTOGRAMXTEST_H_
+#define MANTID_HISTOGRAMDATA_HISTOGRAMXTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidHistogramData/HistogramX.h"
+
+using Mantid::HistogramData::HistogramX;
+
+class HistogramXTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static HistogramXTest *createSuite() { return new HistogramXTest(); }
+  static void destroySuite(HistogramXTest *suite) { delete suite; }
+
+  void test_fake() {}
```

Comment:
Well, most things are tested as parts of the mixins, and I don't think we want to re-test everything (that is another nice feature of the Mixins, we write tests once and use them in many other classes). The only thing to test here are the basic constructors, but they are default, so that is not so useful either.

I think what might make sense is to have a test that verifies if the correct mixins are used. Maybe by a `dynamic_cast` to each of the expected base types (or I can figure out some other method)?

What do you think?